Scripts/GameManagement/GameManager.gd

```diff
@@ -29,7 +30,8 @@ remotesync func initialize_world():
 remotesync func initialize_players():
 	print("initializing players...")
 	for p in LobbyManager.players:
-		var player = Player.instance()
+		var loaded_class = load(CLASSES_PATH + LobbyManager.players[p]["class"] + ".tscn")
```

Comment:
Better name I think would be 'picked_class'

Reply:
I chose this name, because it's a loaded class scene. The information about the picked class is stored in the dictionary.